index.js

```diff
@@ -22,16 +21,28 @@ function cssExtract (bundle, opts) {
   addHooks()
 
   function addHooks () {
-    // run before the "debug" step in browserify pipeline
-    bundle.pipeline.get('debug').unshift(through.obj(write, flush))
     const writeStream = (typeof outFile === 'function')
       ? outFile()
-      : ConcatStream(writeOutFile)
+      : ConcatStream(fs.writeFileSync.bind(fs, outFile))
+
+    // run before the "label" step in browserify pipeline
+    // this makes sure insert-css requires are found before plugins like bundle-collapser run
+    bundle.pipeline.get('label').unshift(through.obj(write, flush))
```

Comment:
Perhaps I'm missing the context, but we should probably discuss this change and its implications in a separate issue if we haven't already :)